samples/tfm/tfm_hello_world/README.rst

```diff
@@ -0,0 +1,32 @@
+.. _tfm_hello_world:
+
+TF-M Hello World
+################
+
+Overview
+********
+
+A simple sample based on Hello World that demonstrates adding TF-M to an application.
+
+This sample uses the PSA API to calculate a SHA256 digest.
+The PSA API call is handled by the TFM secure firmware.
+
+See @ref tfm_doc??? for more information about TFM.
```

Comment:
Missing reference? Not sure what the `???` is for here.

Reply:
I think this was fixed in @b-gent 's commit which should be squashed.